### PR TITLE
Add getter and setter for ROOT_MEAN_SQUARE_THRESHOLD

### LINKED ISSUE #97: Add getter and setter for ROOT_MEAN_SQUARE_THRESHOLD
The current code inside `updateLocation()` is like this 

```
 if (multilateration.getRMS() < ROOT_MEAN_SQUARE_THRESHOLD_STRICT) {
            locationPredictor.addLocation(location);
            onLocationUpdated(getMeanLocation(2, TimeUnit.SECONDS));
        }

```
The user should choose between threshold : `ROOT_MEAN_SQUARE_THRESHOLD_STRICT`, `ROOT_MEAN_SQUARE_THRESHOLD_MEDIUM`, `ROOT_MEAN_SQUARE_THRESHOLD_LIGHT`

Check the PR [here](https://github.com/neXenio/BLE-Indoor-Positioning/pull/96)

### PR DESCRIPTION
**Pull request by hadiidbouk**
Closes #97

### CODE REVIEW

File: BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/IndoorPositioning.java [hunk 1]
```diff
@@ -33,6 +33,7 @@
 
     public static final double MAXIMUM_MOVEMENT_SPEED_NOT_SET = -1;
     private double maximumMovementSpeed = MAXIMUM_MOVEMENT_SPEED_NOT_SET;
+    private int ROOT_MEAN_SQUARE_THRESHOLD = ROOT_MEAN_SQUARE_THRESHOLD_STRICT;
```
Comment (Steppschuh) [inline BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/IndoorPositioning.java]: As the field is not a constant anymore, it should be renamed to `rootMeanSquareThreshold `

File: BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/IndoorPositioning.java [hunk 2]
```diff
@@ -192,4 +193,11 @@ public void setUsableIndoorPositioningBeaconFilter(GenericBeaconFilter usableInd
         this.usableIndoorPositioningBeaconFilter = usableIndoorPositioningBeaconFilter;
     }
 
+    public void setRootMeanSquareThreshold(int rootMeanSquareThreshold) {
+        IndoorPositioning.getInstance().ROOT_MEAN_SQUARE_THRESHOLD = rootMeanSquareThreshold;
```
Comment (Steppschuh) [inline BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/IndoorPositioning.java]: No need to use the static reference `IndoorPositioning.getInstance()` here.

File: BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/IndoorPositioning.java [hunk 3]
```diff
@@ -92,7 +93,7 @@ private void updateLocation() {
 
         // The root mean square of multilateration is used to filter out inaccurate locations.
         // Adjust value to allow location updates with higher deviation
-        if (multilateration.getRMS() < ROOT_MEAN_SQUARE_THRESHOLD_STRICT) {
+        if (multilateration.getRMS() < ROOT_MEAN_SQUARE_THRESHOLD) {
```
Comment (Steppschuh) [inline BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/IndoorPositioning.java]: This should be `rootMeanSquareThreshold `

Comment (hadiidbouk) [reply BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/IndoorPositioning.java]: Sorry for that

**Pull request merged**